.circleci/config.yml

```diff
@@ -13,6 +13,17 @@ jobs:
       - run:
           name: Bundle install
           command: bundle install --path=vendor/bundle
+      - run:
+          name: Check Podfile matches Podfile.lock
+          command: |
+            function helpful_error () {
+              echo "Podfile and Podfile.lock do not match. Please run 'bundle exec pod install' and try again."
+            }
+            trap helpful_error ERR
+
+            # This verifies that the PODFILE CHECKSUM in Podfile.lock matches Podfile
+            PODFILE_SHA1=$(ruby -e "require 'yaml';puts YAML.load_file('Podfile.lock')['PODFILE CHECKSUM']")
```

Comment:
This is not exactly pretty but there is no easy way to parse a YAML file in bash. I don't want to make another script do this so that these steps can be portable to other repos.

Reply:
Works for me!